### PR TITLE
INTYGFV-16524: Added label to question period in CertificateModelFactoryFK7443.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

File: certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverterTest.java [hunk 1]
```diff
@@ -62,6 +63,7 @@ void shouldReturnConvertedConfig() {
                     .hideWorkingHours(true)
                     .previousDateRangeText("TEST TEST")
                     .name("NAME")
+                    .label("LABEL")
                     .dateRanges(
```
Comment (mhornfeldt) [inline certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverterTest.java:67]: We can look into the name. Do we want to change it to more descriptive that it's a level under "name" or have a list of "subtexts"

Comment (mhornfeldt) [reply certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverterTest.java:67]: We did a solution in Mina intyg where we had a list of texts which had an id to show what text it was

Comment (ChrilleSigerhed) [reply certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverterTest.java:70]: Implemented new record called SubText that holds an map of type and text. Maybe this is a better more generic way to handle different types of texts in our model?

Not sure about the name SubText tho, what do you think?

Comment (mhornfeldt) [reply certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverterTest.java:70]: I think we should call it texts/text or similar so it's more general. That means that we could include texts that are not subtexts if we need to in the future (info, alerts etc.). You can then describe that it's a sub text or similar with the id. What do you think?

Comment (ChrilleSigerhed) [reply certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverterTest.java:70]: Changed the field to texts and renamed the record to QuestionTexts to be more generic.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Suggest that we don't introduce QuestionText as of yet. Cant see that it is a generic enough concept that adds value instead of just introducing  attribute for label.

We could consider it again at a later stage if the need present itself.

**Review by mww-sopra-steria — APPROVED** [review]

**Pull request merged**